mips64/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM scratch
+ADD alpine-minirootfs-20210212-armv7.tar.gz /
```

Comment:
I think `armv7` there is a typo?

Reply:
Forgot to save the file after modifying it :-)